lib/test/cdo/test_mailjet.rb

```diff
@@ -342,4 +268,51 @@ def test_add_to_contact_list
 
     MailJet.add_to_contact_list(contact, list_id)
   end
+
+  def test_sending_email_with_template
+    template_name = :email_template_name
+    template_config = 'email_template_config'
+    contact_email = 'contact_email'
+    contact_name = 'contact_name'
+    mailjet_contact = 'mailjet_contact'
```

Comment:
Nit: now that you added the faker gem, we could use it here too.

Reply:
There are no benefits to using it here, as we only need to verify that the correct arguments are passed in the right place and order